pkg/flow/grpc/instances.proto

```diff
@@ -24,6 +24,18 @@ message Instance {
 	string errorCode = 6;
 	string errorMessage = 7;
 	string invoker = 8;
+	Originator originator = 9;
+}
+
```

Comment:
I think having a full-blown edge to the parent and all its information is overkill. And information overload. We should simplify this.

Reply:
agree